src/validation.cpp

```diff
@@ -2749,7 +2692,7 @@ bool CChainState::ActivateBestChainStep(BlockValidationState& state, const CChai
         fBlocksDisconnected = true;
     }
 
-    // Build list of new blocks to connect.
+    // Build list of new blocks to connect (in reverse order).
```

Comment:
Nit, feel free to ignore.
```suggestion
    // Build list of new blocks to connect (in descending height order).
```

Reply:
Picked